src/main/drivers/gyro_sync.h

```diff
@@ -0,0 +1,13 @@
+/*
+ * gyro_sync.h
+ *
+ *  Created on: 3 aug. 2015
+ *      Author: borisb
+ */
+
+extern bool gyroIsUpdated;
```

Comment:
The flag is possibly dangerous - it is cleared after imu update, so it's state in unnecessarily complicated. Using simple bool function is IMO better